imap_processing/hi/hi_cdf_attrs.py

```diff
@@ -158,3 +159,54 @@
     var_type="support_data",
     variable_purpose="PRIMARY",
 )
+
+# Histogram attributes
+hi_hist_l1a_global_attrs = GlobalDataLevelAttrs(
+    data_type="L1A_CNT>Level-1A Histogram",
+    logical_source="imap_hi_l1a_hist",
+    logical_source_desc=("IMAP-HI Instrument Level-1A Histogram Data."),
+    instrument_base=hi_base,
+)
+
+hi_hist_l1a_angle_attrs = AttrBase(
+    validmin=GlobalConstants.INT_FILLVAL,
+    validmax=GlobalConstants.INT_MAXVAL,
+    format="I9",
+    label_axis="ANGLE",
+    display_type="time_series",
+    catdesc=("Angle bin" "Bin centers for Histogram data."),
+    fieldname="ANGLE",
+    fill_val=GlobalConstants.INT_FILLVAL,
+    var_type="support_data",
+)
+
+hi_hist_l1a_esa_step_attrs = ScienceAttrs(
+    validmin=0,
+    validmax=2**4 - 1,
+    depend_0="epoch",
+    format="I2",
+    units="int",
```

Comment:
Are these unitless quantities? I think that is supposed to be left empty in the units category, not the datatype.
```suggestion
    units="",
```

Reply:
Right. I don't know what the units are for this field. I would think that it is some fraction of a second, but am unsure.
@tech3371 With Vivek leaving/off the project who should I ask for clarification about what this value is?

Reply:
Yaxi, yaxi.liu@swri.org

Reply:
I might have resolution on what this field is and what the units should be later today. Other units have been removed.